Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/services/StepService.java

```diff
@@ -57,18 +51,14 @@ public void addStep(StepDTO stepDTO, Long caseID) {
 
     public void removeAllSteps(Long caseId) {
         Case caze = caseDAO.getOne(caseId);
-        List<Step> stepList = caze.getSteps();
 
-        for (Step st: stepList) {
-            stepDAO.delete(st.getId());
-        }
+        caze.getSteps().forEach(step -> stepDAO.delete(step));
+
         caze.setSteps(null);
```

Comment:
Why we need it here?

Reply:
i'm not sure, i will check it, and remove if this set is unnecessary.
 (Ofc i will do it in the "Refactoring" branch, because it's not about StreamAPI)

Reply:
fair enough :)